CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,7 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Perf Alert] Stop raising an alert for the `XCP-ng Tools` SR storage usage (PR [#6755](https://github.com/vatesfr/xen-orchestra/pull/6755))
```

Comment:
What's the source of this change, of this PR?

Reply:
It's this ticket: https://help.vates.fr/#ticket/zoom/12874

Reply:
Why is this alert irrelevant? Why does it make sense to hide it?

Reply:
This storage is always full, so XO will keep raising an alert even though you can't free space on it.

Reply:
I think there is a better way to filter alarms on these kind of SRs.

We should chose one or more conditions between the following:

- `content_type: "iso"`
- `physical_usage: -1`
- `size: -1`
- `SR_type: "iso"`
- `physical_usage: -1`

In our lab I see other SRs that should probably not trigger alarms, like *DVD drives* or *ISOs*. You can see all of them via this (internal) link http://xoa-qa.localdomain/rest/v0/srs?fields=name_label,content_type,SR_type,physical_usage,size

@olivierlambert Any opinions on this?

Reply:
Indeed. In any case, if `size` is 0, I don't think it's relevant to trigger any alarm on that kind of SR. That might be the easiest filter we can choose.